kiosk/src/Services/GamepadManager.ts

```diff
@@ -497,15 +513,11 @@ class GamepadManager {
         gamepad: Gamepad,
         control: GamepadControl
     ): ControlValue {
-        const axisIndex = gamepadControlToPinIndex[control];
-        const threshold =
-            gamepadControlToAxisDirection[control] *
-            configData.GamepadLeftRightThreshold;
-        if (axisIndex < 0 || axisIndex >= gamepad.axes.length) {
-            throw new Error(
-                `Gamepad at index ${gamepad.index} does not have an axis at index ${axisIndex}`
-            );
-        }
+        const axisConf = configData.GamepadAxes[gamepadControlToAxis[control]];
+        const axisIndex = axisConf.Pin;
+        if (axisIndex < 0 || axisIndex >= gamepad.axes.length)
+            return ControlValue.Up;
+        const threshold = axisConf.Sign * axisConf.Threshold;
```

Comment:
What does threshold mean?

Reply:
This value represents the threshold at which a gamepad axis is considered active. Gamepad sticks, even when not touched, rarely report (0,0) as their position due to inherent noise. They're always changing and reporting small, off-center values, like (0.01, -0.003). Furthermore, small stick movements report erratic positions and aren't reliable. The threshold allows us to filter out this noise and only respond to meaningful movements.